packages/design-system/src/MetaButton.tsx

```diff
@@ -16,8 +16,9 @@ export const MetaButton: React.FC<
     fontSize="sm"
     bg="purple.400"
     color="white"
-    {...{ ref }}
-    {...props}
+    transition="background 0.5s, filter 0.75s"
```

Comment:
I would reduce the transition duration to 0.25s and 0.5s. Also is this necessary and/or mentioned by the design team?

Reply:
Removed.